[2024-02-27T12:07:12.509+0700] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: missed_deadline_job_dag.spark_ingestion_to_s3 manual__2024-02-26T09:25:40.972548+00:00 [queued]>
[2024-02-27T12:07:12.536+0700] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: missed_deadline_job_dag.spark_ingestion_to_s3 manual__2024-02-26T09:25:40.972548+00:00 [queued]>
[2024-02-27T12:07:12.545+0700] {taskinstance.py:2170} INFO - Starting attempt 3 of 3
[2024-02-27T12:07:12.585+0700] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): spark_ingestion_to_s3> on 2024-02-26 09:25:40.972548+00:00
[2024-02-27T12:07:12.589+0700] {standard_task_runner.py:60} INFO - Started process 32160 to run task
[2024-02-27T12:07:12.598+0700] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'missed_deadline_job_dag', 'spark_ingestion_to_s3', 'manual__2024-02-26T09:25:40.972548+00:00', '--job-id', '148', '--raw', '--subdir', 'DAGS_FOLDER/missed_deadlinee_dag.py', '--cfg-path', '/tmp/tmpusto2vs4']
[2024-02-27T12:07:12.601+0700] {standard_task_runner.py:88} INFO - Job 148: Subtask spark_ingestion_to_s3
[2024-02-27T12:07:12.796+0700] {task_command.py:423} INFO - Running <TaskInstance: missed_deadline_job_dag.spark_ingestion_to_s3 manual__2024-02-26T09:25:40.972548+00:00 [running]> on host MinhNhut.
[2024-02-27T12:07:13.110+0700] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='missed_deadline_job_dag' AIRFLOW_CTX_TASK_ID='spark_ingestion_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-02-26T09:25:40.972548+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-26T09:25:40.972548+00:00'
[2024-02-27T12:07:13.111+0700] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-02-27T12:07:13.112+0700] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /home/nhut/etl-***/pipeline/spark_ingestion_to_s3.py']
[2024-02-27T12:07:13.126+0700] {subprocess.py:86} INFO - Output:
[2024-02-27T12:07:23.979+0700] {subprocess.py:93} INFO - 24/02/27 12:07:23 WARN Utils: Your hostname, MinhNhut resolves to a loopback address: 127.0.1.1; using 172.28.89.67 instead (on interface eth0)
[2024-02-27T12:07:23.984+0700] {subprocess.py:93} INFO - 24/02/27 12:07:23 WARN Utils: Set SPARK_LOCAL_IP if you need to bind to another address
[2024-02-27T12:07:35.193+0700] {subprocess.py:93} INFO - :: loading settings :: url = jar:file:/home/nhut/etl-***/.venv/lib/python3.10/site-packages/pyspark/jars/ivy-2.5.1.jar!/org/apache/ivy/core/settings/ivysettings.xml
[2024-02-27T12:07:35.822+0700] {subprocess.py:93} INFO - Ivy Default Cache set to: /home/nhut/.ivy2/cache
[2024-02-27T12:07:35.823+0700] {subprocess.py:93} INFO - The jars for the packages stored in: /home/nhut/.ivy2/jars
[2024-02-27T12:07:35.843+0700] {subprocess.py:93} INFO - org.apache.hadoop#hadoop-aws added as a dependency
[2024-02-27T12:07:35.864+0700] {subprocess.py:93} INFO - :: resolving dependencies :: org.apache.spark#spark-submit-parent-8386558c-3b1b-47d9-bdc6-b5bfa114936a;1.0
[2024-02-27T12:07:35.865+0700] {subprocess.py:93} INFO - 	confs: [default]
[2024-02-27T12:07:36.673+0700] {subprocess.py:93} INFO - 	found org.apache.hadoop#hadoop-aws;3.3.2 in spark-list
[2024-02-27T12:07:36.888+0700] {subprocess.py:93} INFO - 	found com.amazonaws#aws-java-sdk-bundle;1.11.1026 in spark-list
[2024-02-27T12:07:37.016+0700] {subprocess.py:93} INFO - 	found org.wildfly.openssl#wildfly-openssl;1.0.7.Final in central
[2024-02-27T12:07:37.133+0700] {subprocess.py:93} INFO - :: resolution report :: resolve 1209ms :: artifacts dl 60ms
[2024-02-27T12:07:37.134+0700] {subprocess.py:93} INFO - 	:: modules in use:
[2024-02-27T12:07:37.135+0700] {subprocess.py:93} INFO - 	com.amazonaws#aws-java-sdk-bundle;1.11.1026 from spark-list in [default]
[2024-02-27T12:07:37.136+0700] {subprocess.py:93} INFO - 	org.apache.hadoop#hadoop-aws;3.3.2 from spark-list in [default]
[2024-02-27T12:07:37.137+0700] {subprocess.py:93} INFO - 	org.wildfly.openssl#wildfly-openssl;1.0.7.Final from central in [default]
[2024-02-27T12:07:37.138+0700] {subprocess.py:93} INFO - 	---------------------------------------------------------------------
[2024-02-27T12:07:37.139+0700] {subprocess.py:93} INFO - 	|                  |            modules            ||   artifacts   |
[2024-02-27T12:07:37.140+0700] {subprocess.py:93} INFO - 	|       conf       | number| search|dwnlded|evicted|| number|dwnlded|
[2024-02-27T12:07:37.141+0700] {subprocess.py:93} INFO - 	---------------------------------------------------------------------
[2024-02-27T12:07:37.143+0700] {subprocess.py:93} INFO - 	|      default     |   3   |   0   |   0   |   0   ||   3   |   0   |
[2024-02-27T12:07:37.143+0700] {subprocess.py:93} INFO - 	---------------------------------------------------------------------
[2024-02-27T12:07:37.154+0700] {subprocess.py:93} INFO - :: retrieving :: org.apache.spark#spark-submit-parent-8386558c-3b1b-47d9-bdc6-b5bfa114936a
[2024-02-27T12:07:37.155+0700] {subprocess.py:93} INFO - 	confs: [default]
[2024-02-27T12:07:37.192+0700] {subprocess.py:93} INFO - 	0 artifacts copied, 3 already retrieved (0kB/16ms)
[2024-02-27T12:07:38.406+0700] {subprocess.py:93} INFO - 24/02/27 12:07:38 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-02-27T12:07:39.762+0700] {subprocess.py:93} INFO - Setting default log level to "WARN".
[2024-02-27T12:07:39.763+0700] {subprocess.py:93} INFO - To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).
[2024-02-27T12:07:45.650+0700] {subprocess.py:93} INFO - 24/02/27 12:07:45 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2024-02-27T12:07:45.655+0700] {subprocess.py:93} INFO - 24/02/27 12:07:45 WARN Utils: Service 'SparkUI' could not bind on port 4041. Attempting port 4042.
[2024-02-27T12:07:45.679+0700] {subprocess.py:93} INFO - 24/02/27 12:07:45 WARN Utils: Service 'SparkUI' could not bind on port 4042. Attempting port 4043.
[2024-02-27T12:07:45.685+0700] {subprocess.py:93} INFO - 24/02/27 12:07:45 WARN Utils: Service 'SparkUI' could not bind on port 4043. Attempting port 4044.
[2024-02-27T12:07:45.694+0700] {subprocess.py:93} INFO - 24/02/27 12:07:45 WARN Utils: Service 'SparkUI' could not bind on port 4044. Attempting port 4045.
[2024-02-27T12:07:45.697+0700] {subprocess.py:93} INFO - 24/02/27 12:07:45 WARN Utils: Service 'SparkUI' could not bind on port 4045. Attempting port 4046.
[2024-02-27T12:07:45.714+0700] {subprocess.py:93} INFO - 24/02/27 12:07:45 WARN Utils: Service 'SparkUI' could not bind on port 4046. Attempting port 4047.
[2024-02-27T12:07:45.717+0700] {subprocess.py:93} INFO - 24/02/27 12:07:45 WARN Utils: Service 'SparkUI' could not bind on port 4047. Attempting port 4048.
[2024-02-27T12:07:45.722+0700] {subprocess.py:93} INFO - 24/02/27 12:07:45 WARN Utils: Service 'SparkUI' could not bind on port 4048. Attempting port 4049.
[2024-02-27T12:08:08.040+0700] {subprocess.py:93} INFO - 24/02/27 12:08:08 ERROR Inbox: Ignoring error
[2024-02-27T12:08:08.055+0700] {subprocess.py:93} INFO - java.lang.NullPointerException
[2024-02-27T12:08:08.058+0700] {subprocess.py:93} INFO - 	at org.apache.spark.storage.BlockManagerMasterEndpoint.org$apache$spark$storage$BlockManagerMasterEndpoint$$register(BlockManagerMasterEndpoint.scala:677)
[2024-02-27T12:08:08.059+0700] {subprocess.py:93} INFO - 	at org.apache.spark.storage.BlockManagerMasterEndpoint$$anonfun$receiveAndReply$1.applyOrElse(BlockManagerMasterEndpoint.scala:133)
[2024-02-27T12:08:08.062+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rpc.netty.Inbox.$anonfun$process$1(Inbox.scala:103)
[2024-02-27T12:08:08.062+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rpc.netty.Inbox.safelyCall(Inbox.scala:213)
[2024-02-27T12:08:08.062+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rpc.netty.Inbox.process(Inbox.scala:100)
[2024-02-27T12:08:08.063+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rpc.netty.MessageLoop.org$apache$spark$rpc$netty$MessageLoop$$receiveLoop(MessageLoop.scala:75)
[2024-02-27T12:08:08.063+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rpc.netty.MessageLoop$$anon$1.run(MessageLoop.scala:41)
[2024-02-27T12:08:08.063+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2024-02-27T12:08:08.063+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2024-02-27T12:08:08.064+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-02-27T12:08:08.064+0700] {subprocess.py:93} INFO - 24/02/27 12:08:08 WARN Executor: Issue communicating with driver in heartbeater
[2024-02-27T12:08:08.064+0700] {subprocess.py:93} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2024-02-27T12:08:08.064+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkThreadUtils$.awaitResult(SparkThreadUtils.scala:56)
[2024-02-27T12:08:08.064+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:310)
[2024-02-27T12:08:08.065+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2024-02-27T12:08:08.065+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rpc.RpcEndpointRef.askSync(RpcEndpointRef.scala:101)
[2024-02-27T12:08:08.065+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rpc.RpcEndpointRef.askSync(RpcEndpointRef.scala:85)
[2024-02-27T12:08:08.065+0700] {subprocess.py:93} INFO - 	at org.apache.spark.storage.BlockManagerMaster.registerBlockManager(BlockManagerMaster.scala:80)
[2024-02-27T12:08:08.065+0700] {subprocess.py:93} INFO - 	at org.apache.spark.storage.BlockManager.reregister(BlockManager.scala:642)
[2024-02-27T12:08:08.066+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor.reportHeartBeat(Executor.scala:1223)
[2024-02-27T12:08:08.066+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor.$anonfun$heartbeater$1(Executor.scala:295)
[2024-02-27T12:08:08.066+0700] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2024-02-27T12:08:08.066+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.logUncaughtExceptions(Utils.scala:1928)
[2024-02-27T12:08:08.067+0700] {subprocess.py:93} INFO - 	at org.apache.spark.Heartbeater$$anon$1.run(Heartbeater.scala:46)
[2024-02-27T12:08:08.067+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2024-02-27T12:08:08.067+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
[2024-02-27T12:08:08.067+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
[2024-02-27T12:08:08.068+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2024-02-27T12:08:08.068+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2024-02-27T12:08:08.068+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-02-27T12:08:08.068+0700] {subprocess.py:93} INFO - Caused by: java.lang.NullPointerException
[2024-02-27T12:08:08.068+0700] {subprocess.py:93} INFO - 	at org.apache.spark.storage.BlockManagerMasterEndpoint.org$apache$spark$storage$BlockManagerMasterEndpoint$$register(BlockManagerMasterEndpoint.scala:677)
[2024-02-27T12:08:08.069+0700] {subprocess.py:93} INFO - 	at org.apache.spark.storage.BlockManagerMasterEndpoint$$anonfun$receiveAndReply$1.applyOrElse(BlockManagerMasterEndpoint.scala:133)
[2024-02-27T12:08:08.069+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rpc.netty.Inbox.$anonfun$process$1(Inbox.scala:103)
[2024-02-27T12:08:08.069+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rpc.netty.Inbox.safelyCall(Inbox.scala:213)
[2024-02-27T12:08:08.069+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rpc.netty.Inbox.process(Inbox.scala:100)
[2024-02-27T12:08:08.070+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rpc.netty.MessageLoop.org$apache$spark$rpc$netty$MessageLoop$$receiveLoop(MessageLoop.scala:75)
[2024-02-27T12:08:08.070+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rpc.netty.MessageLoop$$anon$1.run(MessageLoop.scala:41)
[2024-02-27T12:08:08.070+0700] {subprocess.py:93} INFO - 	... 3 more
[2024-02-27T12:10:58.986+0700] {subprocess.py:93} INFO - 24/02/27 12:10:58 WARN NettyRpcEnv: Ignored failure: java.util.concurrent.TimeoutException: Cannot receive any reply from 172.28.89.67:46067 in 10000 milliseconds
[2024-02-27T12:10:59.080+0700] {subprocess.py:93} INFO - 24/02/27 12:10:58 WARN Executor: Issue communicating with driver in heartbeater
[2024-02-27T12:10:59.080+0700] {subprocess.py:93} INFO - org.apache.spark.rpc.RpcTimeoutException: Futures timed out after [10000 milliseconds]. This timeout is controlled by spark.executor.heartbeatInterval
[2024-02-27T12:10:59.081+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rpc.RpcTimeout.org$apache$spark$rpc$RpcTimeout$$createRpcTimeoutException(RpcTimeout.scala:47)
[2024-02-27T12:10:59.081+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rpc.RpcTimeout$$anonfun$addMessageIfTimeout$1.applyOrElse(RpcTimeout.scala:62)
[2024-02-27T12:10:59.081+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rpc.RpcTimeout$$anonfun$addMessageIfTimeout$1.applyOrElse(RpcTimeout.scala:58)
[2024-02-27T12:10:59.082+0700] {subprocess.py:93} INFO - 	at scala.runtime.AbstractPartialFunction.apply(AbstractPartialFunction.scala:38)
[2024-02-27T12:10:59.082+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:76)
[2024-02-27T12:10:59.082+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rpc.RpcEndpointRef.askSync(RpcEndpointRef.scala:101)
[2024-02-27T12:10:59.083+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor.reportHeartBeat(Executor.scala:1219)
[2024-02-27T12:10:59.083+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor.$anonfun$heartbeater$1(Executor.scala:295)
[2024-02-27T12:10:59.083+0700] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2024-02-27T12:10:59.083+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.logUncaughtExceptions(Utils.scala:1928)
[2024-02-27T12:10:59.084+0700] {subprocess.py:93} INFO - 	at org.apache.spark.Heartbeater$$anon$1.run(Heartbeater.scala:46)
[2024-02-27T12:10:59.084+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2024-02-27T12:10:59.084+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
[2024-02-27T12:10:59.084+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
[2024-02-27T12:10:59.085+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2024-02-27T12:10:59.085+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2024-02-27T12:10:59.086+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-02-27T12:10:59.086+0700] {subprocess.py:93} INFO - Caused by: java.util.concurrent.TimeoutException: Futures timed out after [10000 milliseconds]
[2024-02-27T12:10:59.086+0700] {subprocess.py:93} INFO - 	at scala.concurrent.impl.Promise$DefaultPromise.ready(Promise.scala:259)
[2024-02-27T12:10:59.086+0700] {subprocess.py:93} INFO - 	at scala.concurrent.impl.Promise$DefaultPromise.result(Promise.scala:263)
[2024-02-27T12:10:59.087+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkThreadUtils$.awaitResult(SparkThreadUtils.scala:48)
[2024-02-27T12:10:59.087+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:310)
[2024-02-27T12:10:59.087+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2024-02-27T12:10:59.088+0700] {subprocess.py:93} INFO - 	... 12 more
[2024-02-27T12:12:37.430+0700] {subprocess.py:93} INFO - [Stage 0:>                                                          (0 + 0) / 1][Stage 0:>                                                          (0 + 1) / 1]                                                                                24/02/27 12:12:37 WARN MetricsConfig: Cannot locate configuration: tried hadoop-metrics2-s3a-file-system.properties,hadoop-metrics2.properties
[2024-02-27T12:13:35.612+0700] {subprocess.py:93} INFO - [Stage 1:>                                                          (0 + 4) / 4]24/02/27 12:13:35 ERROR SparkHadoopMapRedUtil: Error committing the output of task: attempt_202402271213059216581647053330429_0001_m_000000_1
[2024-02-27T12:13:35.636+0700] {subprocess.py:93} INFO - java.io.FileNotFoundException: No such file or directory: s3a://olistsalesbucket/olist_order_items_dataset.parquet/_temporary/0/_temporary/attempt_202402271213059216581647053330429_0001_m_000000_1
[2024-02-27T12:13:35.637+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.s3GetFileStatus(S3AFileSystem.java:3866)
[2024-02-27T12:13:35.637+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerGetFileStatus(S3AFileSystem.java:3688)
[2024-02-27T12:13:35.638+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.initiateRename(S3AFileSystem.java:1887)
[2024-02-27T12:13:35.638+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerRename(S3AFileSystem.java:1988)
[2024-02-27T12:13:35.638+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$rename$7(S3AFileSystem.java:1846)
[2024-02-27T12:13:35.638+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2024-02-27T12:13:35.638+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.trackDuration(IOStatisticsBinding.java:444)
[2024-02-27T12:13:35.639+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.trackDurationAndSpan(S3AFileSystem.java:2337)
[2024-02-27T12:13:35.639+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.rename(S3AFileSystem.java:1844)
[2024-02-27T12:13:35.639+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:600)
[2024-02-27T12:13:35.639+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:571)
[2024-02-27T12:13:35.639+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.$anonfun$commitTask$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:35.639+0700] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2024-02-27T12:13:35.639+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:35.639+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.performCommit$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:35.639+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.commitTask(SparkHadoopMapRedUtil.scala:78)
[2024-02-27T12:13:35.639+0700] {subprocess.py:93} INFO - 	at org.apache.spark.internal.io.HadoopMapReduceCommitProtocol.commitTask(HadoopMapReduceCommitProtocol.scala:279)
[2024-02-27T12:13:35.640+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.$anonfun$commit$1(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:35.640+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:35.640+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.commit(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:35.640+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.$anonfun$executeTask$1(FileFormatWriter.scala:404)
[2024-02-27T12:13:35.640+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinallyAndFailureCallbacks(Utils.scala:1397)
[2024-02-27T12:13:35.640+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:410)
[2024-02-27T12:13:35.641+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.WriteFilesExec.$anonfun$doExecuteWrite$1(WriteFiles.scala:100)
[2024-02-27T12:13:35.641+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-02-27T12:13:35.641+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-02-27T12:13:35.641+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-02-27T12:13:35.641+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-02-27T12:13:35.641+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-02-27T12:13:35.641+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-02-27T12:13:35.641+0700] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-02-27T12:13:35.641+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-02-27T12:13:35.641+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-02-27T12:13:35.642+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-02-27T12:13:35.642+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-02-27T12:13:35.642+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-02-27T12:13:35.642+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-02-27T12:13:35.642+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2024-02-27T12:13:35.642+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2024-02-27T12:13:35.642+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-02-27T12:13:35.919+0700] {subprocess.py:93} INFO - 24/02/27 12:13:35 ERROR SparkHadoopMapRedUtil: Error committing the output of task: attempt_202402271213059216581647053330429_0001_m_000002_3
[2024-02-27T12:13:35.919+0700] {subprocess.py:93} INFO - java.io.FileNotFoundException: No such file or directory: s3a://olistsalesbucket/olist_order_items_dataset.parquet/_temporary/0/_temporary/attempt_202402271213059216581647053330429_0001_m_000002_3
[2024-02-27T12:13:35.920+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.s3GetFileStatus(S3AFileSystem.java:3866)
[2024-02-27T12:13:35.920+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerGetFileStatus(S3AFileSystem.java:3688)
[2024-02-27T12:13:35.920+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.initiateRename(S3AFileSystem.java:1887)
[2024-02-27T12:13:35.921+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerRename(S3AFileSystem.java:1988)
[2024-02-27T12:13:35.921+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$rename$7(S3AFileSystem.java:1846)
[2024-02-27T12:13:35.921+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2024-02-27T12:13:35.921+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.trackDuration(IOStatisticsBinding.java:444)
[2024-02-27T12:13:35.922+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.trackDurationAndSpan(S3AFileSystem.java:2337)
[2024-02-27T12:13:35.922+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.rename(S3AFileSystem.java:1844)
[2024-02-27T12:13:35.922+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:600)
[2024-02-27T12:13:35.922+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:571)
[2024-02-27T12:13:35.923+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.$anonfun$commitTask$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:35.923+0700] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2024-02-27T12:13:35.923+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:35.924+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.performCommit$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:35.924+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.commitTask(SparkHadoopMapRedUtil.scala:78)
[2024-02-27T12:13:35.924+0700] {subprocess.py:93} INFO - 	at org.apache.spark.internal.io.HadoopMapReduceCommitProtocol.commitTask(HadoopMapReduceCommitProtocol.scala:279)
[2024-02-27T12:13:35.924+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.$anonfun$commit$1(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:35.925+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:35.925+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.commit(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:35.925+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.$anonfun$executeTask$1(FileFormatWriter.scala:404)
[2024-02-27T12:13:35.925+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinallyAndFailureCallbacks(Utils.scala:1397)
[2024-02-27T12:13:35.926+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:410)
[2024-02-27T12:13:35.926+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.WriteFilesExec.$anonfun$doExecuteWrite$1(WriteFiles.scala:100)
[2024-02-27T12:13:35.926+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-02-27T12:13:35.926+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-02-27T12:13:35.927+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-02-27T12:13:35.927+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-02-27T12:13:35.927+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-02-27T12:13:35.927+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-02-27T12:13:35.927+0700] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-02-27T12:13:35.928+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-02-27T12:13:35.928+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-02-27T12:13:35.928+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-02-27T12:13:35.928+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-02-27T12:13:35.928+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-02-27T12:13:35.929+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-02-27T12:13:35.929+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2024-02-27T12:13:35.929+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2024-02-27T12:13:35.930+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-02-27T12:13:35.930+0700] {subprocess.py:93} INFO - 24/02/27 12:13:35 ERROR SparkHadoopMapRedUtil: Error committing the output of task: attempt_202402271213059216581647053330429_0001_m_000003_4
[2024-02-27T12:13:35.930+0700] {subprocess.py:93} INFO - java.io.FileNotFoundException: No such file or directory: s3a://olistsalesbucket/olist_order_items_dataset.parquet/_temporary/0/_temporary/attempt_202402271213059216581647053330429_0001_m_000003_4
[2024-02-27T12:13:35.931+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.s3GetFileStatus(S3AFileSystem.java:3866)
[2024-02-27T12:13:35.931+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerGetFileStatus(S3AFileSystem.java:3688)
[2024-02-27T12:13:35.931+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.initiateRename(S3AFileSystem.java:1887)
[2024-02-27T12:13:35.931+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerRename(S3AFileSystem.java:1988)
[2024-02-27T12:13:35.932+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$rename$7(S3AFileSystem.java:1846)
[2024-02-27T12:13:35.932+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2024-02-27T12:13:35.932+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.trackDuration(IOStatisticsBinding.java:444)
[2024-02-27T12:13:35.932+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.trackDurationAndSpan(S3AFileSystem.java:2337)
[2024-02-27T12:13:35.933+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.rename(S3AFileSystem.java:1844)
[2024-02-27T12:13:35.933+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:600)
[2024-02-27T12:13:35.933+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:571)
[2024-02-27T12:13:35.933+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.$anonfun$commitTask$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:35.934+0700] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2024-02-27T12:13:35.936+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:35.939+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.performCommit$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:35.940+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.commitTask(SparkHadoopMapRedUtil.scala:78)
[2024-02-27T12:13:35.940+0700] {subprocess.py:93} INFO - 	at org.apache.spark.internal.io.HadoopMapReduceCommitProtocol.commitTask(HadoopMapReduceCommitProtocol.scala:279)
[2024-02-27T12:13:35.940+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.$anonfun$commit$1(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:35.940+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:35.940+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.commit(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:35.940+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.$anonfun$executeTask$1(FileFormatWriter.scala:404)
[2024-02-27T12:13:35.941+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinallyAndFailureCallbacks(Utils.scala:1397)
[2024-02-27T12:13:35.941+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:410)
[2024-02-27T12:13:35.941+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.WriteFilesExec.$anonfun$doExecuteWrite$1(WriteFiles.scala:100)
[2024-02-27T12:13:35.941+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-02-27T12:13:35.941+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-02-27T12:13:35.941+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-02-27T12:13:35.941+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-02-27T12:13:35.942+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-02-27T12:13:35.942+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-02-27T12:13:35.942+0700] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-02-27T12:13:35.942+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-02-27T12:13:35.942+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-02-27T12:13:35.942+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-02-27T12:13:35.943+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-02-27T12:13:35.943+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-02-27T12:13:35.943+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-02-27T12:13:35.943+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2024-02-27T12:13:35.943+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2024-02-27T12:13:35.943+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-02-27T12:13:36.192+0700] {subprocess.py:93} INFO - 24/02/27 12:13:36 WARN FileOutputCommitter: Could not delete s3a://olistsalesbucket/olist_order_items_dataset.parquet/_temporary/0/_temporary/attempt_202402271213059216581647053330429_0001_m_000000_1
[2024-02-27T12:13:36.196+0700] {subprocess.py:93} INFO - 24/02/27 12:13:36 ERROR Utils: Aborting task
[2024-02-27T12:13:36.197+0700] {subprocess.py:93} INFO - java.io.FileNotFoundException: No such file or directory: s3a://olistsalesbucket/olist_order_items_dataset.parquet/_temporary/0/_temporary/attempt_202402271213059216581647053330429_0001_m_000000_1
[2024-02-27T12:13:36.197+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.s3GetFileStatus(S3AFileSystem.java:3866)
[2024-02-27T12:13:36.197+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerGetFileStatus(S3AFileSystem.java:3688)
[2024-02-27T12:13:36.198+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.initiateRename(S3AFileSystem.java:1887)
[2024-02-27T12:13:36.198+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerRename(S3AFileSystem.java:1988)
[2024-02-27T12:13:36.198+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$rename$7(S3AFileSystem.java:1846)
[2024-02-27T12:13:36.198+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2024-02-27T12:13:36.198+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.trackDuration(IOStatisticsBinding.java:444)
[2024-02-27T12:13:36.199+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.trackDurationAndSpan(S3AFileSystem.java:2337)
[2024-02-27T12:13:36.199+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.rename(S3AFileSystem.java:1844)
[2024-02-27T12:13:36.199+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:600)
[2024-02-27T12:13:36.199+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:571)
[2024-02-27T12:13:36.200+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.$anonfun$commitTask$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:36.200+0700] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2024-02-27T12:13:36.200+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:36.200+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.performCommit$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:36.201+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.commitTask(SparkHadoopMapRedUtil.scala:78)
[2024-02-27T12:13:36.201+0700] {subprocess.py:93} INFO - 	at org.apache.spark.internal.io.HadoopMapReduceCommitProtocol.commitTask(HadoopMapReduceCommitProtocol.scala:279)
[2024-02-27T12:13:36.201+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.$anonfun$commit$1(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:36.201+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:36.202+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.commit(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:36.202+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.$anonfun$executeTask$1(FileFormatWriter.scala:404)
[2024-02-27T12:13:36.202+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinallyAndFailureCallbacks(Utils.scala:1397)
[2024-02-27T12:13:36.202+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:410)
[2024-02-27T12:13:36.203+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.WriteFilesExec.$anonfun$doExecuteWrite$1(WriteFiles.scala:100)
[2024-02-27T12:13:36.203+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-02-27T12:13:36.203+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-02-27T12:13:36.203+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-02-27T12:13:36.203+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-02-27T12:13:36.203+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-02-27T12:13:36.204+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-02-27T12:13:36.204+0700] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-02-27T12:13:36.204+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-02-27T12:13:36.204+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-02-27T12:13:36.204+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-02-27T12:13:36.204+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-02-27T12:13:36.204+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-02-27T12:13:36.204+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-02-27T12:13:36.205+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2024-02-27T12:13:36.205+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2024-02-27T12:13:36.205+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-02-27T12:13:36.511+0700] {subprocess.py:93} INFO - 24/02/27 12:13:36 WARN FileOutputCommitter: Could not delete s3a://olistsalesbucket/olist_order_items_dataset.parquet/_temporary/0/_temporary/attempt_202402271213059216581647053330429_0001_m_000003_4
[2024-02-27T12:13:36.512+0700] {subprocess.py:93} INFO - 24/02/27 12:13:36 WARN FileOutputCommitter: Could not delete s3a://olistsalesbucket/olist_order_items_dataset.parquet/_temporary/0/_temporary/attempt_202402271213059216581647053330429_0001_m_000002_3
[2024-02-27T12:13:36.512+0700] {subprocess.py:93} INFO - 24/02/27 12:13:36 ERROR Utils: Aborting task
[2024-02-27T12:13:36.512+0700] {subprocess.py:93} INFO - java.io.FileNotFoundException: No such file or directory: s3a://olistsalesbucket/olist_order_items_dataset.parquet/_temporary/0/_temporary/attempt_202402271213059216581647053330429_0001_m_000003_4
[2024-02-27T12:13:36.512+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.s3GetFileStatus(S3AFileSystem.java:3866)
[2024-02-27T12:13:36.512+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerGetFileStatus(S3AFileSystem.java:3688)
[2024-02-27T12:13:36.513+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.initiateRename(S3AFileSystem.java:1887)
[2024-02-27T12:13:36.513+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerRename(S3AFileSystem.java:1988)
[2024-02-27T12:13:36.513+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$rename$7(S3AFileSystem.java:1846)
[2024-02-27T12:13:36.513+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2024-02-27T12:13:36.513+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.trackDuration(IOStatisticsBinding.java:444)
[2024-02-27T12:13:36.513+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.trackDurationAndSpan(S3AFileSystem.java:2337)
[2024-02-27T12:13:36.513+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.rename(S3AFileSystem.java:1844)
[2024-02-27T12:13:36.514+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:600)
[2024-02-27T12:13:36.514+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:571)
[2024-02-27T12:13:36.514+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.$anonfun$commitTask$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:36.514+0700] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2024-02-27T12:13:36.514+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:36.514+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.performCommit$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:36.514+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.commitTask(SparkHadoopMapRedUtil.scala:78)
[2024-02-27T12:13:36.514+0700] {subprocess.py:93} INFO - 	at org.apache.spark.internal.io.HadoopMapReduceCommitProtocol.commitTask(HadoopMapReduceCommitProtocol.scala:279)
[2024-02-27T12:13:36.515+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.$anonfun$commit$1(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:36.515+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:36.515+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.commit(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:36.515+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.$anonfun$executeTask$1(FileFormatWriter.scala:404)
[2024-02-27T12:13:36.515+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinallyAndFailureCallbacks(Utils.scala:1397)
[2024-02-27T12:13:36.515+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:410)
[2024-02-27T12:13:36.515+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.WriteFilesExec.$anonfun$doExecuteWrite$1(WriteFiles.scala:100)
[2024-02-27T12:13:36.515+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-02-27T12:13:36.515+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-02-27T12:13:36.515+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-02-27T12:13:36.516+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-02-27T12:13:36.516+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-02-27T12:13:36.516+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-02-27T12:13:36.516+0700] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-02-27T12:13:36.516+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-02-27T12:13:36.516+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-02-27T12:13:36.516+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-02-27T12:13:36.516+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-02-27T12:13:36.516+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-02-27T12:13:36.517+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-02-27T12:13:36.517+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2024-02-27T12:13:36.517+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2024-02-27T12:13:36.517+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-02-27T12:13:36.517+0700] {subprocess.py:93} INFO - 24/02/27 12:13:36 ERROR Utils: Aborting task
[2024-02-27T12:13:36.517+0700] {subprocess.py:93} INFO - java.io.FileNotFoundException: No such file or directory: s3a://olistsalesbucket/olist_order_items_dataset.parquet/_temporary/0/_temporary/attempt_202402271213059216581647053330429_0001_m_000002_3
[2024-02-27T12:13:36.518+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.s3GetFileStatus(S3AFileSystem.java:3866)
[2024-02-27T12:13:36.518+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerGetFileStatus(S3AFileSystem.java:3688)
[2024-02-27T12:13:36.518+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.initiateRename(S3AFileSystem.java:1887)
[2024-02-27T12:13:36.519+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerRename(S3AFileSystem.java:1988)
[2024-02-27T12:13:36.519+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$rename$7(S3AFileSystem.java:1846)
[2024-02-27T12:13:36.519+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2024-02-27T12:13:36.519+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.trackDuration(IOStatisticsBinding.java:444)
[2024-02-27T12:13:36.519+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.trackDurationAndSpan(S3AFileSystem.java:2337)
[2024-02-27T12:13:36.520+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.rename(S3AFileSystem.java:1844)
[2024-02-27T12:13:36.520+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:600)
[2024-02-27T12:13:36.520+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:571)
[2024-02-27T12:13:36.520+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.$anonfun$commitTask$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:36.520+0700] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2024-02-27T12:13:36.520+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:36.521+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.performCommit$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:36.521+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.commitTask(SparkHadoopMapRedUtil.scala:78)
[2024-02-27T12:13:36.521+0700] {subprocess.py:93} INFO - 	at org.apache.spark.internal.io.HadoopMapReduceCommitProtocol.commitTask(HadoopMapReduceCommitProtocol.scala:279)
[2024-02-27T12:13:36.521+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.$anonfun$commit$1(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:36.521+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:36.521+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.commit(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:36.522+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.$anonfun$executeTask$1(FileFormatWriter.scala:404)
[2024-02-27T12:13:36.522+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinallyAndFailureCallbacks(Utils.scala:1397)
[2024-02-27T12:13:36.522+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:410)
[2024-02-27T12:13:36.522+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.WriteFilesExec.$anonfun$doExecuteWrite$1(WriteFiles.scala:100)
[2024-02-27T12:13:36.523+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-02-27T12:13:36.523+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-02-27T12:13:36.523+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-02-27T12:13:36.523+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-02-27T12:13:36.524+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-02-27T12:13:36.524+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-02-27T12:13:36.524+0700] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-02-27T12:13:36.524+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-02-27T12:13:36.525+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-02-27T12:13:36.525+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-02-27T12:13:36.525+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-02-27T12:13:36.525+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-02-27T12:13:36.526+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-02-27T12:13:36.526+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2024-02-27T12:13:36.526+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2024-02-27T12:13:36.526+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-02-27T12:13:36.782+0700] {subprocess.py:93} INFO - 24/02/27 12:13:36 WARN FileOutputCommitter: Could not delete s3a://olistsalesbucket/olist_order_items_dataset.parquet/_temporary/0/_temporary/attempt_202402271213059216581647053330429_0001_m_000000_1
[2024-02-27T12:13:36.783+0700] {subprocess.py:93} INFO - 24/02/27 12:13:36 ERROR FileFormatWriter: Job job_202402271213059216581647053330429_0001 aborted.
[2024-02-27T12:13:36.898+0700] {subprocess.py:93} INFO - 24/02/27 12:13:36 ERROR Executor: Exception in task 0.0 in stage 1.0 (TID 1)
[2024-02-27T12:13:36.898+0700] {subprocess.py:93} INFO - org.apache.spark.SparkException: [TASK_WRITE_FAILED] Task failed while writing rows to s3a://olistsalesbucket/olist_order_items_dataset.parquet.
[2024-02-27T12:13:36.899+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.taskFailedWhileWritingRowsError(QueryExecutionErrors.scala:774)
[2024-02-27T12:13:36.899+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:420)
[2024-02-27T12:13:36.899+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.WriteFilesExec.$anonfun$doExecuteWrite$1(WriteFiles.scala:100)
[2024-02-27T12:13:36.899+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-02-27T12:13:36.899+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-02-27T12:13:36.900+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-02-27T12:13:36.900+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-02-27T12:13:36.900+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-02-27T12:13:36.900+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-02-27T12:13:36.900+0700] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-02-27T12:13:36.900+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-02-27T12:13:36.900+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-02-27T12:13:36.900+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-02-27T12:13:36.900+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-02-27T12:13:36.901+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-02-27T12:13:36.901+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-02-27T12:13:36.901+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2024-02-27T12:13:36.901+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2024-02-27T12:13:36.901+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-02-27T12:13:36.901+0700] {subprocess.py:93} INFO - Caused by: java.io.FileNotFoundException: No such file or directory: s3a://olistsalesbucket/olist_order_items_dataset.parquet/_temporary/0/_temporary/attempt_202402271213059216581647053330429_0001_m_000000_1
[2024-02-27T12:13:36.901+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.s3GetFileStatus(S3AFileSystem.java:3866)
[2024-02-27T12:13:36.901+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerGetFileStatus(S3AFileSystem.java:3688)
[2024-02-27T12:13:36.901+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.initiateRename(S3AFileSystem.java:1887)
[2024-02-27T12:13:36.901+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerRename(S3AFileSystem.java:1988)
[2024-02-27T12:13:36.901+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$rename$7(S3AFileSystem.java:1846)
[2024-02-27T12:13:36.902+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2024-02-27T12:13:36.902+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.trackDuration(IOStatisticsBinding.java:444)
[2024-02-27T12:13:36.902+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.trackDurationAndSpan(S3AFileSystem.java:2337)
[2024-02-27T12:13:36.902+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.rename(S3AFileSystem.java:1844)
[2024-02-27T12:13:36.902+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:600)
[2024-02-27T12:13:36.902+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:571)
[2024-02-27T12:13:36.902+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.$anonfun$commitTask$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:36.902+0700] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2024-02-27T12:13:36.902+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:36.902+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.performCommit$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:36.903+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.commitTask(SparkHadoopMapRedUtil.scala:78)
[2024-02-27T12:13:36.903+0700] {subprocess.py:93} INFO - 	at org.apache.spark.internal.io.HadoopMapReduceCommitProtocol.commitTask(HadoopMapReduceCommitProtocol.scala:279)
[2024-02-27T12:13:36.903+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.$anonfun$commit$1(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:36.903+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:36.903+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.commit(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:36.903+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.$anonfun$executeTask$1(FileFormatWriter.scala:404)
[2024-02-27T12:13:36.903+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinallyAndFailureCallbacks(Utils.scala:1397)
[2024-02-27T12:13:36.903+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:410)
[2024-02-27T12:13:36.903+0700] {subprocess.py:93} INFO - 	... 17 more
[2024-02-27T12:13:36.961+0700] {subprocess.py:93} INFO - 24/02/27 12:13:36 WARN TaskSetManager: Lost task 0.0 in stage 1.0 (TID 1) (172.28.89.67 executor driver): org.apache.spark.SparkException: [TASK_WRITE_FAILED] Task failed while writing rows to s3a://olistsalesbucket/olist_order_items_dataset.parquet.
[2024-02-27T12:13:36.963+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.taskFailedWhileWritingRowsError(QueryExecutionErrors.scala:774)
[2024-02-27T12:13:36.964+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:420)
[2024-02-27T12:13:36.964+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.WriteFilesExec.$anonfun$doExecuteWrite$1(WriteFiles.scala:100)
[2024-02-27T12:13:36.965+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-02-27T12:13:36.965+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-02-27T12:13:36.965+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-02-27T12:13:36.965+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-02-27T12:13:36.966+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-02-27T12:13:36.966+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-02-27T12:13:36.966+0700] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-02-27T12:13:36.966+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-02-27T12:13:36.966+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-02-27T12:13:36.966+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-02-27T12:13:36.967+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-02-27T12:13:36.967+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-02-27T12:13:36.967+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-02-27T12:13:36.967+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2024-02-27T12:13:36.967+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2024-02-27T12:13:36.967+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-02-27T12:13:36.967+0700] {subprocess.py:93} INFO - Caused by: java.io.FileNotFoundException: No such file or directory: s3a://olistsalesbucket/olist_order_items_dataset.parquet/_temporary/0/_temporary/attempt_202402271213059216581647053330429_0001_m_000000_1
[2024-02-27T12:13:36.967+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.s3GetFileStatus(S3AFileSystem.java:3866)
[2024-02-27T12:13:36.967+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerGetFileStatus(S3AFileSystem.java:3688)
[2024-02-27T12:13:36.967+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.initiateRename(S3AFileSystem.java:1887)
[2024-02-27T12:13:36.968+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerRename(S3AFileSystem.java:1988)
[2024-02-27T12:13:36.968+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$rename$7(S3AFileSystem.java:1846)
[2024-02-27T12:13:36.968+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2024-02-27T12:13:36.968+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.trackDuration(IOStatisticsBinding.java:444)
[2024-02-27T12:13:36.968+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.trackDurationAndSpan(S3AFileSystem.java:2337)
[2024-02-27T12:13:36.968+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.rename(S3AFileSystem.java:1844)
[2024-02-27T12:13:36.968+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:600)
[2024-02-27T12:13:36.968+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:571)
[2024-02-27T12:13:36.968+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.$anonfun$commitTask$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:36.968+0700] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2024-02-27T12:13:36.969+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:36.969+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.performCommit$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:36.969+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.commitTask(SparkHadoopMapRedUtil.scala:78)
[2024-02-27T12:13:36.969+0700] {subprocess.py:93} INFO - 	at org.apache.spark.internal.io.HadoopMapReduceCommitProtocol.commitTask(HadoopMapReduceCommitProtocol.scala:279)
[2024-02-27T12:13:36.969+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.$anonfun$commit$1(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:36.969+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:36.969+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.commit(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:36.969+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.$anonfun$executeTask$1(FileFormatWriter.scala:404)
[2024-02-27T12:13:36.969+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinallyAndFailureCallbacks(Utils.scala:1397)
[2024-02-27T12:13:36.969+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:410)
[2024-02-27T12:13:36.969+0700] {subprocess.py:93} INFO - 	... 17 more
[2024-02-27T12:13:36.969+0700] {subprocess.py:93} INFO - 
[2024-02-27T12:13:36.970+0700] {subprocess.py:93} INFO - 24/02/27 12:13:36 ERROR TaskSetManager: Task 0 in stage 1.0 failed 1 times; aborting job
[2024-02-27T12:13:36.975+0700] {subprocess.py:93} INFO - 24/02/27 12:13:36 ERROR FileFormatWriter: Aborting job 27d5e94c-a572-46f4-b41c-f2b075a9fce3.
[2024-02-27T12:13:36.976+0700] {subprocess.py:93} INFO - org.apache.spark.SparkException: Job aborted due to stage failure: Task 0 in stage 1.0 failed 1 times, most recent failure: Lost task 0.0 in stage 1.0 (TID 1) (172.28.89.67 executor driver): org.apache.spark.SparkException: [TASK_WRITE_FAILED] Task failed while writing rows to s3a://olistsalesbucket/olist_order_items_dataset.parquet.
[2024-02-27T12:13:36.976+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.taskFailedWhileWritingRowsError(QueryExecutionErrors.scala:774)
[2024-02-27T12:13:36.976+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:420)
[2024-02-27T12:13:36.976+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.WriteFilesExec.$anonfun$doExecuteWrite$1(WriteFiles.scala:100)
[2024-02-27T12:13:36.976+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-02-27T12:13:36.976+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-02-27T12:13:36.976+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-02-27T12:13:36.977+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-02-27T12:13:36.977+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-02-27T12:13:36.977+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-02-27T12:13:36.977+0700] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-02-27T12:13:36.977+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-02-27T12:13:36.977+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-02-27T12:13:36.977+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-02-27T12:13:36.977+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-02-27T12:13:36.977+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-02-27T12:13:36.978+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-02-27T12:13:36.978+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2024-02-27T12:13:36.978+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2024-02-27T12:13:36.978+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-02-27T12:13:36.978+0700] {subprocess.py:93} INFO - Caused by: java.io.FileNotFoundException: No such file or directory: s3a://olistsalesbucket/olist_order_items_dataset.parquet/_temporary/0/_temporary/attempt_202402271213059216581647053330429_0001_m_000000_1
[2024-02-27T12:13:36.978+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.s3GetFileStatus(S3AFileSystem.java:3866)
[2024-02-27T12:13:36.978+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerGetFileStatus(S3AFileSystem.java:3688)
[2024-02-27T12:13:36.978+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.initiateRename(S3AFileSystem.java:1887)
[2024-02-27T12:13:36.978+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerRename(S3AFileSystem.java:1988)
[2024-02-27T12:13:36.978+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$rename$7(S3AFileSystem.java:1846)
[2024-02-27T12:13:36.978+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2024-02-27T12:13:36.979+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.trackDuration(IOStatisticsBinding.java:444)
[2024-02-27T12:13:36.979+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.trackDurationAndSpan(S3AFileSystem.java:2337)
[2024-02-27T12:13:36.979+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.rename(S3AFileSystem.java:1844)
[2024-02-27T12:13:36.979+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:600)
[2024-02-27T12:13:36.979+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:571)
[2024-02-27T12:13:36.979+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.$anonfun$commitTask$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:36.980+0700] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2024-02-27T12:13:36.980+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:36.980+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.performCommit$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:36.980+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.commitTask(SparkHadoopMapRedUtil.scala:78)
[2024-02-27T12:13:36.980+0700] {subprocess.py:93} INFO - 	at org.apache.spark.internal.io.HadoopMapReduceCommitProtocol.commitTask(HadoopMapReduceCommitProtocol.scala:279)
[2024-02-27T12:13:36.980+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.$anonfun$commit$1(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:36.980+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:36.980+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.commit(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:36.981+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.$anonfun$executeTask$1(FileFormatWriter.scala:404)
[2024-02-27T12:13:36.981+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinallyAndFailureCallbacks(Utils.scala:1397)
[2024-02-27T12:13:36.982+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:410)
[2024-02-27T12:13:36.982+0700] {subprocess.py:93} INFO - 	... 17 more
[2024-02-27T12:13:36.982+0700] {subprocess.py:93} INFO - 
[2024-02-27T12:13:36.982+0700] {subprocess.py:93} INFO - Driver stacktrace:
[2024-02-27T12:13:36.982+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.failJobAndIndependentStages(DAGScheduler.scala:2856)
[2024-02-27T12:13:36.983+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.$anonfun$abortStage$2(DAGScheduler.scala:2792)
[2024-02-27T12:13:36.983+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.$anonfun$abortStage$2$adapted(DAGScheduler.scala:2791)
[2024-02-27T12:13:36.983+0700] {subprocess.py:93} INFO - 	at scala.collection.mutable.ResizableArray.foreach(ResizableArray.scala:62)
[2024-02-27T12:13:36.983+0700] {subprocess.py:93} INFO - 	at scala.collection.mutable.ResizableArray.foreach$(ResizableArray.scala:55)
[2024-02-27T12:13:36.983+0700] {subprocess.py:93} INFO - 	at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:49)
[2024-02-27T12:13:36.983+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.abortStage(DAGScheduler.scala:2791)
[2024-02-27T12:13:36.984+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.$anonfun$handleTaskSetFailed$1(DAGScheduler.scala:1247)
[2024-02-27T12:13:36.984+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.$anonfun$handleTaskSetFailed$1$adapted(DAGScheduler.scala:1247)
[2024-02-27T12:13:36.984+0700] {subprocess.py:93} INFO - 	at scala.Option.foreach(Option.scala:407)
[2024-02-27T12:13:36.984+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.handleTaskSetFailed(DAGScheduler.scala:1247)
[2024-02-27T12:13:36.984+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.doOnReceive(DAGScheduler.scala:3060)
[2024-02-27T12:13:36.984+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.onReceive(DAGScheduler.scala:2994)
[2024-02-27T12:13:36.984+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.onReceive(DAGScheduler.scala:2983)
[2024-02-27T12:13:36.984+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.EventLoop$$anon$1.run(EventLoop.scala:49)
[2024-02-27T12:13:36.984+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.runJob(DAGScheduler.scala:989)
[2024-02-27T12:13:36.985+0700] {subprocess.py:93} INFO - 	at org.apache.spark.SparkContext.runJob(SparkContext.scala:2398)
[2024-02-27T12:13:36.985+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.$anonfun$executeWrite$4(FileFormatWriter.scala:307)
[2024-02-27T12:13:36.985+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.writeAndCommit(FileFormatWriter.scala:271)
[2024-02-27T12:13:36.985+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeWrite(FileFormatWriter.scala:304)
[2024-02-27T12:13:36.985+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.write(FileFormatWriter.scala:190)
[2024-02-27T12:13:36.985+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.InsertIntoHadoopFsRelationCommand.run(InsertIntoHadoopFsRelationCommand.scala:190)
[2024-02-27T12:13:36.985+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.command.DataWritingCommandExec.sideEffectResult$lzycompute(commands.scala:113)
[2024-02-27T12:13:36.985+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.command.DataWritingCommandExec.sideEffectResult(commands.scala:111)
[2024-02-27T12:13:36.985+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.command.DataWritingCommandExec.executeCollect(commands.scala:125)
[2024-02-27T12:13:36.986+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution$$anonfun$eagerlyExecuteCommands$1.$anonfun$applyOrElse$1(QueryExecution.scala:107)
[2024-02-27T12:13:36.986+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.$anonfun$withNewExecutionId$6(SQLExecution.scala:125)
[2024-02-27T12:13:36.986+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.withSQLConfPropagated(SQLExecution.scala:201)
[2024-02-27T12:13:36.986+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.$anonfun$withNewExecutionId$1(SQLExecution.scala:108)
[2024-02-27T12:13:36.987+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.SparkSession.withActive(SparkSession.scala:900)
[2024-02-27T12:13:36.987+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.withNewExecutionId(SQLExecution.scala:66)
[2024-02-27T12:13:36.987+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution$$anonfun$eagerlyExecuteCommands$1.applyOrElse(QueryExecution.scala:107)
[2024-02-27T12:13:36.987+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution$$anonfun$eagerlyExecuteCommands$1.applyOrElse(QueryExecution.scala:98)
[2024-02-27T12:13:36.987+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.trees.TreeNode.$anonfun$transformDownWithPruning$1(TreeNode.scala:461)
[2024-02-27T12:13:36.987+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.trees.CurrentOrigin$.withOrigin(origin.scala:76)
[2024-02-27T12:13:36.988+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.trees.TreeNode.transformDownWithPruning(TreeNode.scala:461)
[2024-02-27T12:13:36.988+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.LogicalPlan.org$apache$spark$sql$catalyst$plans$logical$AnalysisHelper$$super$transformDownWithPruning(LogicalPlan.scala:32)
[2024-02-27T12:13:36.988+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.AnalysisHelper.transformDownWithPruning(AnalysisHelper.scala:267)
[2024-02-27T12:13:36.988+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.AnalysisHelper.transformDownWithPruning$(AnalysisHelper.scala:263)
[2024-02-27T12:13:36.988+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.LogicalPlan.transformDownWithPruning(LogicalPlan.scala:32)
[2024-02-27T12:13:36.988+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.LogicalPlan.transformDownWithPruning(LogicalPlan.scala:32)
[2024-02-27T12:13:36.988+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.trees.TreeNode.transformDown(TreeNode.scala:437)
[2024-02-27T12:13:36.988+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution.eagerlyExecuteCommands(QueryExecution.scala:98)
[2024-02-27T12:13:36.988+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution.commandExecuted$lzycompute(QueryExecution.scala:85)
[2024-02-27T12:13:36.988+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution.commandExecuted(QueryExecution.scala:83)
[2024-02-27T12:13:36.989+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution.assertCommandExecuted(QueryExecution.scala:142)
[2024-02-27T12:13:36.989+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameWriter.runCommand(DataFrameWriter.scala:859)
[2024-02-27T12:13:36.989+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameWriter.saveToV1Source(DataFrameWriter.scala:388)
[2024-02-27T12:13:36.989+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameWriter.saveInternal(DataFrameWriter.scala:361)
[2024-02-27T12:13:36.989+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameWriter.save(DataFrameWriter.scala:240)
[2024-02-27T12:13:36.989+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameWriter.parquet(DataFrameWriter.scala:792)
[2024-02-27T12:13:36.989+0700] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2024-02-27T12:13:36.989+0700] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[2024-02-27T12:13:36.989+0700] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-02-27T12:13:36.989+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[2024-02-27T12:13:36.990+0700] {subprocess.py:93} INFO - 	at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
[2024-02-27T12:13:36.990+0700] {subprocess.py:93} INFO - 	at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-02-27T12:13:36.990+0700] {subprocess.py:93} INFO - 	at py4j.Gateway.invoke(Gateway.java:282)
[2024-02-27T12:13:36.990+0700] {subprocess.py:93} INFO - 	at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
[2024-02-27T12:13:36.990+0700] {subprocess.py:93} INFO - 	at py4j.commands.CallCommand.execute(CallCommand.java:79)
[2024-02-27T12:13:36.990+0700] {subprocess.py:93} INFO - 	at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-02-27T12:13:36.990+0700] {subprocess.py:93} INFO - 	at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-02-27T12:13:36.990+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-02-27T12:13:36.990+0700] {subprocess.py:93} INFO - Caused by: org.apache.spark.SparkException: [TASK_WRITE_FAILED] Task failed while writing rows to s3a://olistsalesbucket/olist_order_items_dataset.parquet.
[2024-02-27T12:13:36.991+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.taskFailedWhileWritingRowsError(QueryExecutionErrors.scala:774)
[2024-02-27T12:13:36.991+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:420)
[2024-02-27T12:13:36.991+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.WriteFilesExec.$anonfun$doExecuteWrite$1(WriteFiles.scala:100)
[2024-02-27T12:13:36.991+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-02-27T12:13:36.991+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-02-27T12:13:36.992+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-02-27T12:13:36.992+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-02-27T12:13:36.992+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-02-27T12:13:36.992+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-02-27T12:13:36.992+0700] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-02-27T12:13:36.992+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-02-27T12:13:36.992+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-02-27T12:13:36.992+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-02-27T12:13:36.992+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-02-27T12:13:36.993+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-02-27T12:13:37.006+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-02-27T12:13:37.006+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2024-02-27T12:13:37.006+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2024-02-27T12:13:37.006+0700] {subprocess.py:93} INFO - 	... 1 more
[2024-02-27T12:13:37.007+0700] {subprocess.py:93} INFO - Caused by: java.io.FileNotFoundException: No such file or directory: s3a://olistsalesbucket/olist_order_items_dataset.parquet/_temporary/0/_temporary/attempt_202402271213059216581647053330429_0001_m_000000_1
[2024-02-27T12:13:37.007+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.s3GetFileStatus(S3AFileSystem.java:3866)
[2024-02-27T12:13:37.007+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerGetFileStatus(S3AFileSystem.java:3688)
[2024-02-27T12:13:37.008+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.initiateRename(S3AFileSystem.java:1887)
[2024-02-27T12:13:37.008+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerRename(S3AFileSystem.java:1988)
[2024-02-27T12:13:37.008+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$rename$7(S3AFileSystem.java:1846)
[2024-02-27T12:13:37.008+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2024-02-27T12:13:37.008+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.trackDuration(IOStatisticsBinding.java:444)
[2024-02-27T12:13:37.008+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.trackDurationAndSpan(S3AFileSystem.java:2337)
[2024-02-27T12:13:37.009+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.rename(S3AFileSystem.java:1844)
[2024-02-27T12:13:37.009+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:600)
[2024-02-27T12:13:37.009+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:571)
[2024-02-27T12:13:37.009+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.$anonfun$commitTask$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:37.009+0700] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2024-02-27T12:13:37.009+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:37.010+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.performCommit$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:37.010+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.commitTask(SparkHadoopMapRedUtil.scala:78)
[2024-02-27T12:13:37.010+0700] {subprocess.py:93} INFO - 	at org.apache.spark.internal.io.HadoopMapReduceCommitProtocol.commitTask(HadoopMapReduceCommitProtocol.scala:279)
[2024-02-27T12:13:37.010+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.$anonfun$commit$1(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:37.010+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:37.010+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.commit(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:37.010+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.$anonfun$executeTask$1(FileFormatWriter.scala:404)
[2024-02-27T12:13:37.010+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinallyAndFailureCallbacks(Utils.scala:1397)
[2024-02-27T12:13:37.010+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:410)
[2024-02-27T12:13:37.010+0700] {subprocess.py:93} INFO - 	... 17 more
[2024-02-27T12:13:37.101+0700] {subprocess.py:93} INFO - [Stage 1:>                                                          (0 + 3) / 4]24/02/27 12:13:37 WARN FileOutputCommitter: Could not delete s3a://olistsalesbucket/olist_order_items_dataset.parquet/_temporary/0/_temporary/attempt_202402271213059216581647053330429_0001_m_000002_3
[2024-02-27T12:13:37.101+0700] {subprocess.py:93} INFO - 24/02/27 12:13:37 WARN FileOutputCommitter: Could not delete s3a://olistsalesbucket/olist_order_items_dataset.parquet/_temporary/0/_temporary/attempt_202402271213059216581647053330429_0001_m_000003_4
[2024-02-27T12:13:37.102+0700] {subprocess.py:93} INFO - 24/02/27 12:13:37 ERROR FileFormatWriter: Job job_202402271213059216581647053330429_0001 aborted.
[2024-02-27T12:13:37.102+0700] {subprocess.py:93} INFO - 24/02/27 12:13:37 ERROR FileFormatWriter: Job job_202402271213059216581647053330429_0001 aborted.
[2024-02-27T12:13:37.110+0700] {subprocess.py:93} INFO - 24/02/27 12:13:37 WARN TaskSetManager: Lost task 3.0 in stage 1.0 (TID 4) (172.28.89.67 executor driver): TaskKilled (Stage cancelled: Job aborted due to stage failure: Task 0 in stage 1.0 failed 1 times, most recent failure: Lost task 0.0 in stage 1.0 (TID 1) (172.28.89.67 executor driver): org.apache.spark.SparkException: [TASK_WRITE_FAILED] Task failed while writing rows to s3a://olistsalesbucket/olist_order_items_dataset.parquet.
[2024-02-27T12:13:37.111+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.taskFailedWhileWritingRowsError(QueryExecutionErrors.scala:774)
[2024-02-27T12:13:37.111+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:420)
[2024-02-27T12:13:37.111+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.WriteFilesExec.$anonfun$doExecuteWrite$1(WriteFiles.scala:100)
[2024-02-27T12:13:37.111+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-02-27T12:13:37.112+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-02-27T12:13:37.112+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-02-27T12:13:37.112+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-02-27T12:13:37.112+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-02-27T12:13:37.112+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-02-27T12:13:37.112+0700] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-02-27T12:13:37.113+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-02-27T12:13:37.113+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-02-27T12:13:37.113+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-02-27T12:13:37.113+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-02-27T12:13:37.113+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-02-27T12:13:37.113+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-02-27T12:13:37.113+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2024-02-27T12:13:37.113+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2024-02-27T12:13:37.113+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-02-27T12:13:37.113+0700] {subprocess.py:93} INFO - Caused by: java.io.FileNotFoundException: No such file or directory: s3a://olistsalesbucket/olist_order_items_dataset.parquet/_temporary/0/_temporary/attempt_202402271213059216581647053330429_0001_m_000000_1
[2024-02-27T12:13:37.114+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.s3GetFileStatus(S3AFileSystem.java:3866)
[2024-02-27T12:13:37.114+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerGetFileStatus(S3AFileSystem.java:3688)
[2024-02-27T12:13:37.114+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.initiateRename(S3AFileSystem.java:1887)
[2024-02-27T12:13:37.114+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerRename(S3AFileSystem.java:1988)
[2024-02-27T12:13:37.114+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$rename$7(S3AFileSystem.java:1846)
[2024-02-27T12:13:37.114+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2024-02-27T12:13:37.114+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.trackDuration(IOStatisticsBinding.java:444)
[2024-02-27T12:13:37.114+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.trackDurationAndSpan(S3AFileSystem.java:2337)
[2024-02-27T12:13:37.114+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.rename(S3AFileSystem.java:1844)
[2024-02-27T12:13:37.115+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:600)
[2024-02-27T12:13:37.115+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:571)
[2024-02-27T12:13:37.115+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.$anonfun$commitTask$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:37.115+0700] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2024-02-27T12:13:37.115+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:37.115+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.performCommit$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:37.115+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.commitTask(SparkHadoopMapRedUtil.scala:78)
[2024-02-27T12:13:37.115+0700] {subprocess.py:93} INFO - 	at org.apache.spark.internal.io.HadoopMapReduceCommitProtocol.commitTask(HadoopMapReduceCommitProtocol.scala:279)
[2024-02-27T12:13:37.115+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.$anonfun$commit$1(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:37.116+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:37.116+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.commit(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:37.116+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.$anonfun$executeTask$1(FileFormatWriter.scala:404)
[2024-02-27T12:13:37.116+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinallyAndFailureCallbacks(Utils.scala:1397)
[2024-02-27T12:13:37.116+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:410)
[2024-02-27T12:13:37.116+0700] {subprocess.py:93} INFO - 	... 17 more
[2024-02-27T12:13:37.116+0700] {subprocess.py:93} INFO - 
[2024-02-27T12:13:37.116+0700] {subprocess.py:93} INFO - Driver stacktrace:)
[2024-02-27T12:13:37.116+0700] {subprocess.py:93} INFO - 24/02/27 12:13:37 WARN TaskSetManager: Lost task 2.0 in stage 1.0 (TID 3) (172.28.89.67 executor driver): TaskKilled (Stage cancelled: Job aborted due to stage failure: Task 0 in stage 1.0 failed 1 times, most recent failure: Lost task 0.0 in stage 1.0 (TID 1) (172.28.89.67 executor driver): org.apache.spark.SparkException: [TASK_WRITE_FAILED] Task failed while writing rows to s3a://olistsalesbucket/olist_order_items_dataset.parquet.
[2024-02-27T12:13:37.117+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.taskFailedWhileWritingRowsError(QueryExecutionErrors.scala:774)
[2024-02-27T12:13:37.117+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:420)
[2024-02-27T12:13:37.117+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.WriteFilesExec.$anonfun$doExecuteWrite$1(WriteFiles.scala:100)
[2024-02-27T12:13:37.117+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-02-27T12:13:37.117+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-02-27T12:13:37.117+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-02-27T12:13:37.117+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-02-27T12:13:37.117+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-02-27T12:13:37.117+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-02-27T12:13:37.118+0700] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-02-27T12:13:37.118+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-02-27T12:13:37.118+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-02-27T12:13:37.118+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-02-27T12:13:37.118+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-02-27T12:13:37.118+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-02-27T12:13:37.118+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-02-27T12:13:37.118+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2024-02-27T12:13:37.118+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2024-02-27T12:13:37.119+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-02-27T12:13:37.119+0700] {subprocess.py:93} INFO - Caused by: java.io.FileNotFoundException: No such file or directory: s3a://olistsalesbucket/olist_order_items_dataset.parquet/_temporary/0/_temporary/attempt_202402271213059216581647053330429_0001_m_000000_1
[2024-02-27T12:13:37.119+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.s3GetFileStatus(S3AFileSystem.java:3866)
[2024-02-27T12:13:37.119+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerGetFileStatus(S3AFileSystem.java:3688)
[2024-02-27T12:13:37.119+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.initiateRename(S3AFileSystem.java:1887)
[2024-02-27T12:13:37.119+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerRename(S3AFileSystem.java:1988)
[2024-02-27T12:13:37.119+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$rename$7(S3AFileSystem.java:1846)
[2024-02-27T12:13:37.119+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2024-02-27T12:13:37.119+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.trackDuration(IOStatisticsBinding.java:444)
[2024-02-27T12:13:37.120+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.trackDurationAndSpan(S3AFileSystem.java:2337)
[2024-02-27T12:13:37.120+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.rename(S3AFileSystem.java:1844)
[2024-02-27T12:13:37.120+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:600)
[2024-02-27T12:13:37.120+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:571)
[2024-02-27T12:13:37.120+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.$anonfun$commitTask$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:37.120+0700] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2024-02-27T12:13:37.120+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:37.120+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.performCommit$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:37.120+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.commitTask(SparkHadoopMapRedUtil.scala:78)
[2024-02-27T12:13:37.121+0700] {subprocess.py:93} INFO - 	at org.apache.spark.internal.io.HadoopMapReduceCommitProtocol.commitTask(HadoopMapReduceCommitProtocol.scala:279)
[2024-02-27T12:13:37.121+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.$anonfun$commit$1(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:37.121+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:37.121+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.commit(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:37.121+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.$anonfun$executeTask$1(FileFormatWriter.scala:404)
[2024-02-27T12:13:37.121+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinallyAndFailureCallbacks(Utils.scala:1397)
[2024-02-27T12:13:37.121+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:410)
[2024-02-27T12:13:37.121+0700] {subprocess.py:93} INFO - 	... 17 more
[2024-02-27T12:13:37.121+0700] {subprocess.py:93} INFO - 
[2024-02-27T12:13:37.122+0700] {subprocess.py:93} INFO - Driver stacktrace:)
[2024-02-27T12:13:37.597+0700] {subprocess.py:93} INFO - [Stage 1:>                                                          (0 + 1) / 4]24/02/27 12:13:37 WARN TaskSetManager: Lost task 1.0 in stage 1.0 (TID 2) (172.28.89.67 executor driver): TaskKilled (Stage cancelled: Job aborted due to stage failure: Task 0 in stage 1.0 failed 1 times, most recent failure: Lost task 0.0 in stage 1.0 (TID 1) (172.28.89.67 executor driver): org.apache.spark.SparkException: [TASK_WRITE_FAILED] Task failed while writing rows to s3a://olistsalesbucket/olist_order_items_dataset.parquet.
[2024-02-27T12:13:37.597+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.taskFailedWhileWritingRowsError(QueryExecutionErrors.scala:774)
[2024-02-27T12:13:37.598+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:420)
[2024-02-27T12:13:37.598+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.WriteFilesExec.$anonfun$doExecuteWrite$1(WriteFiles.scala:100)
[2024-02-27T12:13:37.598+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-02-27T12:13:37.598+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-02-27T12:13:37.598+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-02-27T12:13:37.598+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-02-27T12:13:37.598+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-02-27T12:13:37.598+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-02-27T12:13:37.599+0700] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-02-27T12:13:37.599+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-02-27T12:13:37.599+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-02-27T12:13:37.599+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-02-27T12:13:37.599+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-02-27T12:13:37.599+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-02-27T12:13:37.599+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-02-27T12:13:37.599+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2024-02-27T12:13:37.599+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2024-02-27T12:13:37.599+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-02-27T12:13:37.599+0700] {subprocess.py:93} INFO - Caused by: java.io.FileNotFoundException: No such file or directory: s3a://olistsalesbucket/olist_order_items_dataset.parquet/_temporary/0/_temporary/attempt_202402271213059216581647053330429_0001_m_000000_1
[2024-02-27T12:13:37.600+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.s3GetFileStatus(S3AFileSystem.java:3866)
[2024-02-27T12:13:37.600+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerGetFileStatus(S3AFileSystem.java:3688)
[2024-02-27T12:13:37.600+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.initiateRename(S3AFileSystem.java:1887)
[2024-02-27T12:13:37.600+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerRename(S3AFileSystem.java:1988)
[2024-02-27T12:13:37.600+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$rename$7(S3AFileSystem.java:1846)
[2024-02-27T12:13:37.600+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2024-02-27T12:13:37.600+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.trackDuration(IOStatisticsBinding.java:444)
[2024-02-27T12:13:37.600+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.trackDurationAndSpan(S3AFileSystem.java:2337)
[2024-02-27T12:13:37.600+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.rename(S3AFileSystem.java:1844)
[2024-02-27T12:13:37.600+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:600)
[2024-02-27T12:13:37.600+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:571)
[2024-02-27T12:13:37.601+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.$anonfun$commitTask$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:37.601+0700] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2024-02-27T12:13:37.601+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:37.601+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.performCommit$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:37.601+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.commitTask(SparkHadoopMapRedUtil.scala:78)
[2024-02-27T12:13:37.601+0700] {subprocess.py:93} INFO - 	at org.apache.spark.internal.io.HadoopMapReduceCommitProtocol.commitTask(HadoopMapReduceCommitProtocol.scala:279)
[2024-02-27T12:13:37.601+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.$anonfun$commit$1(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:37.601+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:37.601+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.commit(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:37.601+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.$anonfun$executeTask$1(FileFormatWriter.scala:404)
[2024-02-27T12:13:37.602+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinallyAndFailureCallbacks(Utils.scala:1397)
[2024-02-27T12:13:37.602+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:410)
[2024-02-27T12:13:37.602+0700] {subprocess.py:93} INFO - 	... 17 more
[2024-02-27T12:13:37.602+0700] {subprocess.py:93} INFO - 
[2024-02-27T12:13:37.602+0700] {subprocess.py:93} INFO - Driver stacktrace:)
[2024-02-27T12:13:39.442+0700] {subprocess.py:93} INFO - Traceback (most recent call last):
[2024-02-27T12:13:39.442+0700] {subprocess.py:93} INFO -   File "/home/nhut/etl-***/pipeline/spark_ingestion_to_s3.py", line 31, in <module>
[2024-02-27T12:13:39.446+0700] {subprocess.py:93} INFO -     extract_data()
[2024-02-27T12:13:39.446+0700] {subprocess.py:93} INFO -   File "/home/nhut/etl-***/pipeline/spark_ingestion_to_s3.py", line 27, in extract_data
[2024-02-27T12:13:39.446+0700] {subprocess.py:93} INFO -     df.write.mode("overwrite").parquet("s3a://olistsalesbucket/" + name_file + ".parquet")
[2024-02-27T12:13:39.446+0700] {subprocess.py:93} INFO -   File "/home/nhut/etl-***/.venv/lib/python3.10/site-packages/pyspark/sql/readwriter.py", line 1721, in parquet
[2024-02-27T12:13:39.447+0700] {subprocess.py:93} INFO -     self._jwrite.parquet(path)
[2024-02-27T12:13:39.447+0700] {subprocess.py:93} INFO -   File "/home/nhut/etl-***/.venv/lib/python3.10/site-packages/py4j/java_gateway.py", line 1322, in __call__
[2024-02-27T12:13:39.447+0700] {subprocess.py:93} INFO -     return_value = get_return_value(
[2024-02-27T12:13:39.447+0700] {subprocess.py:93} INFO -   File "/home/nhut/etl-***/.venv/lib/python3.10/site-packages/pyspark/errors/exceptions/captured.py", line 179, in deco
[2024-02-27T12:13:39.447+0700] {subprocess.py:93} INFO -     return f(*a, **kw)
[2024-02-27T12:13:39.447+0700] {subprocess.py:93} INFO -   File "/home/nhut/etl-***/.venv/lib/python3.10/site-packages/py4j/protocol.py", line 326, in get_return_value
[2024-02-27T12:13:39.447+0700] {subprocess.py:93} INFO -     raise Py4JJavaError(
[2024-02-27T12:13:39.492+0700] {subprocess.py:93} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling o35.parquet.
[2024-02-27T12:13:39.493+0700] {subprocess.py:93} INFO - : org.apache.spark.SparkException: Job aborted due to stage failure: Task 0 in stage 1.0 failed 1 times, most recent failure: Lost task 0.0 in stage 1.0 (TID 1) (172.28.89.67 executor driver): org.apache.spark.SparkException: [TASK_WRITE_FAILED] Task failed while writing rows to s3a://olistsalesbucket/olist_order_items_dataset.parquet.
[2024-02-27T12:13:39.493+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.taskFailedWhileWritingRowsError(QueryExecutionErrors.scala:774)
[2024-02-27T12:13:39.493+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:420)
[2024-02-27T12:13:39.493+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.WriteFilesExec.$anonfun$doExecuteWrite$1(WriteFiles.scala:100)
[2024-02-27T12:13:39.493+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-02-27T12:13:39.493+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-02-27T12:13:39.493+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-02-27T12:13:39.493+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-02-27T12:13:39.494+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-02-27T12:13:39.494+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-02-27T12:13:39.494+0700] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-02-27T12:13:39.494+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-02-27T12:13:39.494+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-02-27T12:13:39.494+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-02-27T12:13:39.494+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-02-27T12:13:39.494+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-02-27T12:13:39.494+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-02-27T12:13:39.494+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2024-02-27T12:13:39.495+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2024-02-27T12:13:39.495+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-02-27T12:13:39.495+0700] {subprocess.py:93} INFO - Caused by: java.io.FileNotFoundException: No such file or directory: s3a://olistsalesbucket/olist_order_items_dataset.parquet/_temporary/0/_temporary/attempt_202402271213059216581647053330429_0001_m_000000_1
[2024-02-27T12:13:39.495+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.s3GetFileStatus(S3AFileSystem.java:3866)
[2024-02-27T12:13:39.495+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerGetFileStatus(S3AFileSystem.java:3688)
[2024-02-27T12:13:39.495+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.initiateRename(S3AFileSystem.java:1887)
[2024-02-27T12:13:39.495+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerRename(S3AFileSystem.java:1988)
[2024-02-27T12:13:39.495+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$rename$7(S3AFileSystem.java:1846)
[2024-02-27T12:13:39.495+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2024-02-27T12:13:39.495+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.trackDuration(IOStatisticsBinding.java:444)
[2024-02-27T12:13:39.495+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.trackDurationAndSpan(S3AFileSystem.java:2337)
[2024-02-27T12:13:39.496+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.rename(S3AFileSystem.java:1844)
[2024-02-27T12:13:39.496+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:600)
[2024-02-27T12:13:39.496+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:571)
[2024-02-27T12:13:39.496+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.$anonfun$commitTask$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:39.496+0700] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2024-02-27T12:13:39.496+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:39.496+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.performCommit$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:39.496+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.commitTask(SparkHadoopMapRedUtil.scala:78)
[2024-02-27T12:13:39.496+0700] {subprocess.py:93} INFO - 	at org.apache.spark.internal.io.HadoopMapReduceCommitProtocol.commitTask(HadoopMapReduceCommitProtocol.scala:279)
[2024-02-27T12:13:39.496+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.$anonfun$commit$1(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:39.496+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:39.497+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.commit(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:39.497+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.$anonfun$executeTask$1(FileFormatWriter.scala:404)
[2024-02-27T12:13:39.497+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinallyAndFailureCallbacks(Utils.scala:1397)
[2024-02-27T12:13:39.497+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:410)
[2024-02-27T12:13:39.497+0700] {subprocess.py:93} INFO - 	... 17 more
[2024-02-27T12:13:39.497+0700] {subprocess.py:93} INFO - 
[2024-02-27T12:13:39.497+0700] {subprocess.py:93} INFO - Driver stacktrace:
[2024-02-27T12:13:39.497+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.failJobAndIndependentStages(DAGScheduler.scala:2856)
[2024-02-27T12:13:39.497+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.$anonfun$abortStage$2(DAGScheduler.scala:2792)
[2024-02-27T12:13:39.498+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.$anonfun$abortStage$2$adapted(DAGScheduler.scala:2791)
[2024-02-27T12:13:39.498+0700] {subprocess.py:93} INFO - 	at scala.collection.mutable.ResizableArray.foreach(ResizableArray.scala:62)
[2024-02-27T12:13:39.498+0700] {subprocess.py:93} INFO - 	at scala.collection.mutable.ResizableArray.foreach$(ResizableArray.scala:55)
[2024-02-27T12:13:39.498+0700] {subprocess.py:93} INFO - 	at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:49)
[2024-02-27T12:13:39.498+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.abortStage(DAGScheduler.scala:2791)
[2024-02-27T12:13:39.498+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.$anonfun$handleTaskSetFailed$1(DAGScheduler.scala:1247)
[2024-02-27T12:13:39.498+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.$anonfun$handleTaskSetFailed$1$adapted(DAGScheduler.scala:1247)
[2024-02-27T12:13:39.498+0700] {subprocess.py:93} INFO - 	at scala.Option.foreach(Option.scala:407)
[2024-02-27T12:13:39.498+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.handleTaskSetFailed(DAGScheduler.scala:1247)
[2024-02-27T12:13:39.498+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.doOnReceive(DAGScheduler.scala:3060)
[2024-02-27T12:13:39.499+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.onReceive(DAGScheduler.scala:2994)
[2024-02-27T12:13:39.499+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.onReceive(DAGScheduler.scala:2983)
[2024-02-27T12:13:39.499+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.EventLoop$$anon$1.run(EventLoop.scala:49)
[2024-02-27T12:13:39.499+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.runJob(DAGScheduler.scala:989)
[2024-02-27T12:13:39.499+0700] {subprocess.py:93} INFO - 	at org.apache.spark.SparkContext.runJob(SparkContext.scala:2398)
[2024-02-27T12:13:39.500+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.$anonfun$executeWrite$4(FileFormatWriter.scala:307)
[2024-02-27T12:13:39.500+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.writeAndCommit(FileFormatWriter.scala:271)
[2024-02-27T12:13:39.500+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeWrite(FileFormatWriter.scala:304)
[2024-02-27T12:13:39.500+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.write(FileFormatWriter.scala:190)
[2024-02-27T12:13:39.500+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.InsertIntoHadoopFsRelationCommand.run(InsertIntoHadoopFsRelationCommand.scala:190)
[2024-02-27T12:13:39.500+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.command.DataWritingCommandExec.sideEffectResult$lzycompute(commands.scala:113)
[2024-02-27T12:13:39.500+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.command.DataWritingCommandExec.sideEffectResult(commands.scala:111)
[2024-02-27T12:13:39.500+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.command.DataWritingCommandExec.executeCollect(commands.scala:125)
[2024-02-27T12:13:39.500+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution$$anonfun$eagerlyExecuteCommands$1.$anonfun$applyOrElse$1(QueryExecution.scala:107)
[2024-02-27T12:13:39.500+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.$anonfun$withNewExecutionId$6(SQLExecution.scala:125)
[2024-02-27T12:13:39.501+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.withSQLConfPropagated(SQLExecution.scala:201)
[2024-02-27T12:13:39.501+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.$anonfun$withNewExecutionId$1(SQLExecution.scala:108)
[2024-02-27T12:13:39.501+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.SparkSession.withActive(SparkSession.scala:900)
[2024-02-27T12:13:39.501+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.withNewExecutionId(SQLExecution.scala:66)
[2024-02-27T12:13:39.501+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution$$anonfun$eagerlyExecuteCommands$1.applyOrElse(QueryExecution.scala:107)
[2024-02-27T12:13:39.501+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution$$anonfun$eagerlyExecuteCommands$1.applyOrElse(QueryExecution.scala:98)
[2024-02-27T12:13:39.501+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.trees.TreeNode.$anonfun$transformDownWithPruning$1(TreeNode.scala:461)
[2024-02-27T12:13:39.501+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.trees.CurrentOrigin$.withOrigin(origin.scala:76)
[2024-02-27T12:13:39.501+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.trees.TreeNode.transformDownWithPruning(TreeNode.scala:461)
[2024-02-27T12:13:39.501+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.LogicalPlan.org$apache$spark$sql$catalyst$plans$logical$AnalysisHelper$$super$transformDownWithPruning(LogicalPlan.scala:32)
[2024-02-27T12:13:39.501+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.AnalysisHelper.transformDownWithPruning(AnalysisHelper.scala:267)
[2024-02-27T12:13:39.502+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.AnalysisHelper.transformDownWithPruning$(AnalysisHelper.scala:263)
[2024-02-27T12:13:39.502+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.LogicalPlan.transformDownWithPruning(LogicalPlan.scala:32)
[2024-02-27T12:13:39.502+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.LogicalPlan.transformDownWithPruning(LogicalPlan.scala:32)
[2024-02-27T12:13:39.502+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.trees.TreeNode.transformDown(TreeNode.scala:437)
[2024-02-27T12:13:39.502+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution.eagerlyExecuteCommands(QueryExecution.scala:98)
[2024-02-27T12:13:39.502+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution.commandExecuted$lzycompute(QueryExecution.scala:85)
[2024-02-27T12:13:39.502+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution.commandExecuted(QueryExecution.scala:83)
[2024-02-27T12:13:39.502+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.QueryExecution.assertCommandExecuted(QueryExecution.scala:142)
[2024-02-27T12:13:39.503+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameWriter.runCommand(DataFrameWriter.scala:859)
[2024-02-27T12:13:39.503+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameWriter.saveToV1Source(DataFrameWriter.scala:388)
[2024-02-27T12:13:39.503+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameWriter.saveInternal(DataFrameWriter.scala:361)
[2024-02-27T12:13:39.503+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameWriter.save(DataFrameWriter.scala:240)
[2024-02-27T12:13:39.503+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameWriter.parquet(DataFrameWriter.scala:792)
[2024-02-27T12:13:39.503+0700] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2024-02-27T12:13:39.503+0700] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[2024-02-27T12:13:39.503+0700] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-02-27T12:13:39.503+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[2024-02-27T12:13:39.504+0700] {subprocess.py:93} INFO - 	at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
[2024-02-27T12:13:39.504+0700] {subprocess.py:93} INFO - 	at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-02-27T12:13:39.504+0700] {subprocess.py:93} INFO - 	at py4j.Gateway.invoke(Gateway.java:282)
[2024-02-27T12:13:39.504+0700] {subprocess.py:93} INFO - 	at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
[2024-02-27T12:13:39.504+0700] {subprocess.py:93} INFO - 	at py4j.commands.CallCommand.execute(CallCommand.java:79)
[2024-02-27T12:13:39.504+0700] {subprocess.py:93} INFO - 	at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-02-27T12:13:39.504+0700] {subprocess.py:93} INFO - 	at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-02-27T12:13:39.504+0700] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2024-02-27T12:13:39.504+0700] {subprocess.py:93} INFO - Caused by: org.apache.spark.SparkException: [TASK_WRITE_FAILED] Task failed while writing rows to s3a://olistsalesbucket/olist_order_items_dataset.parquet.
[2024-02-27T12:13:39.504+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.taskFailedWhileWritingRowsError(QueryExecutionErrors.scala:774)
[2024-02-27T12:13:39.505+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:420)
[2024-02-27T12:13:39.505+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.WriteFilesExec.$anonfun$doExecuteWrite$1(WriteFiles.scala:100)
[2024-02-27T12:13:39.505+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-02-27T12:13:39.505+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-02-27T12:13:39.505+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-02-27T12:13:39.505+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-02-27T12:13:39.505+0700] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-02-27T12:13:39.505+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-02-27T12:13:39.505+0700] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-02-27T12:13:39.505+0700] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-02-27T12:13:39.505+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-02-27T12:13:39.506+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-02-27T12:13:39.506+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-02-27T12:13:39.506+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-02-27T12:13:39.506+0700] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-02-27T12:13:39.506+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2024-02-27T12:13:39.506+0700] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2024-02-27T12:13:39.506+0700] {subprocess.py:93} INFO - 	... 1 more
[2024-02-27T12:13:39.506+0700] {subprocess.py:93} INFO - Caused by: java.io.FileNotFoundException: No such file or directory: s3a://olistsalesbucket/olist_order_items_dataset.parquet/_temporary/0/_temporary/attempt_202402271213059216581647053330429_0001_m_000000_1
[2024-02-27T12:13:39.506+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.s3GetFileStatus(S3AFileSystem.java:3866)
[2024-02-27T12:13:39.506+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerGetFileStatus(S3AFileSystem.java:3688)
[2024-02-27T12:13:39.507+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.initiateRename(S3AFileSystem.java:1887)
[2024-02-27T12:13:39.507+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerRename(S3AFileSystem.java:1988)
[2024-02-27T12:13:39.507+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$rename$7(S3AFileSystem.java:1846)
[2024-02-27T12:13:39.507+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2024-02-27T12:13:39.507+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.trackDuration(IOStatisticsBinding.java:444)
[2024-02-27T12:13:39.507+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.trackDurationAndSpan(S3AFileSystem.java:2337)
[2024-02-27T12:13:39.507+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.rename(S3AFileSystem.java:1844)
[2024-02-27T12:13:39.507+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:600)
[2024-02-27T12:13:39.507+0700] {subprocess.py:93} INFO - 	at org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter.commitTask(FileOutputCommitter.java:571)
[2024-02-27T12:13:39.507+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.$anonfun$commitTask$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:39.508+0700] {subprocess.py:93} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2024-02-27T12:13:39.508+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:39.508+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.performCommit$1(SparkHadoopMapRedUtil.scala:51)
[2024-02-27T12:13:39.508+0700] {subprocess.py:93} INFO - 	at org.apache.spark.mapred.SparkHadoopMapRedUtil$.commitTask(SparkHadoopMapRedUtil.scala:78)
[2024-02-27T12:13:39.508+0700] {subprocess.py:93} INFO - 	at org.apache.spark.internal.io.HadoopMapReduceCommitProtocol.commitTask(HadoopMapReduceCommitProtocol.scala:279)
[2024-02-27T12:13:39.508+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.$anonfun$commit$1(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:39.508+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.timeTakenMs(Utils.scala:552)
[2024-02-27T12:13:39.508+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatDataWriter.commit(FileFormatDataWriter.scala:107)
[2024-02-27T12:13:39.508+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.$anonfun$executeTask$1(FileFormatWriter.scala:404)
[2024-02-27T12:13:39.509+0700] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinallyAndFailureCallbacks(Utils.scala:1397)
[2024-02-27T12:13:39.509+0700] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileFormatWriter$.executeTask(FileFormatWriter.scala:410)
[2024-02-27T12:13:39.509+0700] {subprocess.py:93} INFO - 	... 17 more
[2024-02-27T12:13:39.509+0700] {subprocess.py:93} INFO - 
[2024-02-27T12:13:40.276+0700] {subprocess.py:97} INFO - Command exited with return code 1
[2024-02-27T12:13:40.541+0700] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/nhut/etl-airflow/.venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/nhut/etl-airflow/.venv/lib/python3.10/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-02-27T12:13:40.585+0700] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=missed_deadline_job_dag, task_id=spark_ingestion_to_s3, execution_date=20240226T092540, start_date=20240227T050712, end_date=20240227T051340
[2024-02-27T12:13:40.777+0700] {standard_task_runner.py:107} ERROR - Failed to execute job 148 for task spark_ingestion_to_s3 (Bash command failed. The command returned a non-zero exit code 1.; 32160)
[2024-02-27T12:13:40.829+0700] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-27T12:13:41.115+0700] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
